packages/material-ui-lab/src/themeAugmentation/overrides.d.ts

```diff
@@ -41,29 +22,11 @@ import { YearPickerClassKey } from '../YearPicker';
 export interface LabComponentNameToClassKey {
   MuiCalendarPicker: CalendarPickerClassKey;
   MuiCalendarPickerSkeleton: CalendarPickerSkeletonClassKey;
-  MuiClock: ClockClassKey;
-  MuiClockNumber: ClockNumberClassKey;
-  MuiClockPointer: ClockPointerClassKey;
-  MuiDatePickerToolbar: DatePickerToolbarClassKey;
+  MuiClockPicker: ClockPickerClassKey;
   MuiDateRangePickerDay: DateRangePickerDayClassKey;
-  MuiDateTimePickerTabs: DateTimePickerTabsClassKey;
-  MuiDateTimePickerToolbar: DateTimePickerToolbarClassKey;
   MuiLoadingButton: LoadingButtonClassKey;
   MuiMonthPicker: MonthPickerClassKey;
-  MuiPicker: PickerClassKey;
-  MuiPickersArrowSwitcher: PickersArrowSwitcherClassKey;
-  MuiPickersCalendar: PickersCalendarClassKey;
-  MuiPickersCalendarHeader: PickersCalendarHeaderClassKey;
   MuiPickersDay: PickersDayClassKey;
-  MuiPickersFadeTransition: PickersFadeTransitionGroupClassKey;
-  MuiPickersModalDialog: PickersModalDialogClassKey;
-  MuiPickersMonth: PickersMonthClassKey;
-  MuiPickersPopper: PickersPopperClassKey;
-  MuiPickersSlideTransition: PickersSlideTransitionClassKey;
-  MuiPickersToolbar: PickersToolbarClassKey;
-  MuiPickersToolbarButton: PickersToolbarButtonClassKey;
-  MuiPickersToolbarText: PickersToolbarTextClassKey;
-  MuiPickersYear: PickersYearClassKey;
   MuiTabList: TabListClassKey;
   MuiTabPanel: TabPanelClassKey;
```

Comment:
@mnajdova since we have changed the theme structure, should we drop this file? (and props.d.ts)

Reply:
Aren't they are still used in the theme augmentation, how can we drop them? As far as I can see in https://github.com/eps1lon/material-ui/blob/next/packages/material-ui-lab/src/themeAugmentation/components.d.ts we still depend on theme extending the `overrides.d.ts`

Reply:
Ah yes right. I wonder if having them under the same file could be clearer. It seems daunting to keep these 3 files in sync.

Reply:
Good point, let me see how it would look like if we change it.